contents/forms/button-group/basic/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { Button, ButtonGroup } from "@yamada-ui/react"
+import type { FC } from "react"
+
+const Basic: FC = () => {
```

Comment:
@nanikasi 
![badge](<https://img.shields.io/badge/review-q-blue.svg>)
![badge](<https://img.shields.io/badge/review-imo-orange.svg>)

I think `"BasicButtonGroup"` is clearer as a naming convention, what do you think?

Reply:
@zonoryo17 
Thank you for your review. The reason I used "basic" as the name is that I followed TailwindCSS's example from this link: https://tailwindui.com/components/application-ui/elements/button-groups. However, "BasicButtonGroup" is clearer compared to other components. I will change "Basic" to "BasicButtonGroup" and also update the directory name accordingly. Thank you.